<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="Draw" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="7">
       <widget class="QLineEdit" name="lineEdit_density">
        <property name="text">
         <string>1000</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>kg/mÂ³</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Density</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Tank (gravity)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wind Tunnel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Paint</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wind Tunnel (high resolution)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="checkBox_tracer">
        <property name="text">
         <string>Tracer</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBox_scalars">
        <item>
         <property name="text">
          <string>&lt;Scalar&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pressure</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Velocity magnitude</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X-Velocity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Y-Velocity</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="lineEdit_resolution">
        <property name="text">
         <string>50</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_pause">
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="checkBox_streamlines">
        <property name="text">
         <string>Streamlines</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QCheckBox" name="checkBox_velocityVectors">
        <property name="text">
         <string>Velocity vectors</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QCheckBox" name="checkBox_overrelaxation">
        <property name="text">
         <string>Overrelaxation</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QLineEdit" name="lineEdit_overrelaxation">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>1.9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="checkBox_objectposition">
        <property name="text">
         <string>Object position</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_obstacles">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>&lt;Obstacles&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cylinder</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Square</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Diamond</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NACA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rotor</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1174</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Draw</class>
   <extends>QWidget</extends>
   <header location="global">draw.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_overrelaxation</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit_overrelaxation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>833</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>981</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
